samples/bluetooth/handsfree_ag/Kconfig

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+mainmenu "Bluetooth: handsfree AG"
+
+config BT_HFP_AG_DISCOVER_RESULT_COUNT
+	int "Maximum result count per device discovery"
+	default 10
+
+config BT_HFP_AG_CALL_OUTGOING
+	bool "The simulate call: outgoing (y), incoming (n)"
+	default n
```

Comment:
`n` is the "default default" so you can just remove this line

Reply:
updated